start

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
This will fail if this script is ever run in an environment where `bash` is not in `/bin` (e.g. NixOS). Consider `#!/usr/bin/env bash`, which should work regardless of where the `bash` program is located. Generally a good practice, IMO.

Reply:
Ah great point. Thanks!